op-batcher/batcher/channel_builder.go

```diff
@@ -82,7 +83,15 @@ func (cc *ChannelConfig) Check() error {
 	// will infinitely loop when trying to create frames in the
 	// [channelBuilder.OutputFrames] function.
 	if cc.MaxFrameSize == 0 {
-		return ErrInvalidMaxFrameSize
+		return ErrZeroMaxFrameSize
+	}
+
+	// If the [MaxFrameSize] is set to < 23, the channel out
+	// will underflow the maxSize variable in the [derive.ChannelOut].
+	// Since it is of type uint64, it will wrap around to a very large
+	// number, making the frame size extremely large.
+	if cc.MaxFrameSize < 23 {
```

Comment:
Should be a constant rather than a magic number. In particular this number could change if we update the frame format to a new version.